### ISSUE TITLE
`imports` property is required in the interface type `BundleOptions`

### ISSUE THREAD

**Issue by johnspurlock** [denoland/deno_emit#21]
should it be?

**Comment by johnspurlock** [comment]
Also it fails when calling ` await bundle(rootSpecifier, { imports: {} });`

```
error: Uncaught (in promise) RuntimeError: memory access out of bounds
    at dlmalloc::dlmalloc::Chunk::size::h21e6256dacef1d17 (https://deno.land/x/emit@0.0.1/lib/deno_emit_bg.wasm:1:3790959)
    at __rdl_realloc (https://deno.land/x/emit@0.0.1/lib/deno_emit_bg.wasm:1:1391601)
    at __rust_realloc (https://deno.land/x/emit@0.0.1/lib/deno_emit_bg.wasm:1:3716800)
    at __wbindgen_realloc (https://deno.land/x/emit@0.0.1/lib/deno_emit_bg.wasm:1:3586796)
    at passStringToWasm0 (https://deno.land/x/emit@0.0.1/lib/deno_emit.generated.js:92:11)
    at bundle (https://deno.land/x/emit@0.0.1/lib/deno_emit.generated.js:175:50)
    at bundle (https://deno.land/x/emit@0.0.1/mod.ts:83:10)
```

**Comment by kitsonk** [comment]
Closed in #15
